dev_env/bootstrap.sh

```
@@ -89,30 +89,52 @@ init_pyenv_shell() {
+  if ! have_cmd pyenv; then
+    echo "[aviso] pyenv nao disponivel; criando fallback .venv com python do sistema"
+    if ! have_cmd python3; then
+      echo "[erro] python3 nao encontrado para criar fallback .venv"
+      exit 1
+    fi
+    if ! python3 -m venv --help >/dev/null 2>&1; then
+      echo "[erro] modulo venv indisponivel no python3 atual"
+      exit 1
+    fi
+    python3 -m venv .venv
+    source .venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)